semmc-aarch32/test/TestMain.hs

```diff
@@ -55,8 +55,8 @@ data Flag = N | Z | C | V | Q
   deriving (Show, Eq)
 
 -- | Given a list of flags initializes a CPSR set to user mode
-mkCPSR :: [Flag] -> Word32
-mkCPSR flags = foldr (.|.) 16 [n,z,c,v,q]
+_mkCPSR :: [Flag] -> Word32
```

Comment:
This is unrelated to the main thrust of the PR, but is there any reason to have this `mkCPSR` function (and associated `Flag` data type) here at all? It's certainly not used in the module, and I don't see why it should belong in the test suite driver. I'd favor just removing it.

Reply:
I'm OK with that, I don't really know what these functions are/were for.